frontend/apps/app/components/AppBar/AppBar.tsx

```diff
@@ -1,5 +1,4 @@
-'use client'
-
+import { getProject } from '@/features/projects/pages/ProjectDetailPage/getProject'
```

Comment:
Since the columns to select are different, the getProject function for this component should be defined here